pkg/registry/apis/provisioning/jobs/migrate/storage.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 func stopReadingUnifiedStorage(ctx context.Context, dual dualwrite.Service) error {
-	for _, gr := range resources.SupportedResources {
+	for _, gr := range resources.StandardResources {
```

Comment:
naming... I don't like this name but I understand why you changed it. 


This was my thinking: 

The list contains the resources we support in provisioning and the variable is located in /registry/apis/provisioning/resources -> `SupportedResources`. 


That list contains the resources we can import, sync, comment, render, etc. 


`SupportedProvisioningResources` ?